### PR TITLE
LRQA-29748 Set the user notification timeout to 1 second

### PR DESCRIPTION
**Pull request by alexmds2004**
ticket
https://issues.liferay.com/browse/LRQA-29748

Discription

SPAIfrastructure#setTimeout

### CODE REVIEW

**Comment by alexmds2004** [comment]
ci:test:sf

**Comment by alexmds2004** [comment]
ci:test:sf

**Comment by alexmds2004** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/frontenddataset/SPAInfrastructure.testcase [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+			ServerAdministration.openServerAdmin();
+
+			ServerAdministration.executeScript(
+				language = "Groovy",
+				script = "sleep(1000); out.println('sample output text');");
+		}
+
+		task ("And When: pause for 1 second") {
+			Pause(locator1 = "1000");
+		}
```
Comment (mateusralv) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/frontenddataset/SPAInfrastructure.testcase:59]: Before a Pause, you need to put a comment explaining why you need to use this.

**Comment by alexmds2004** [comment]
ci:test:sf

**Comment by alexmds2004** [comment]
ci:test:sf

**Comment by alexmds2004** [comment]
@mateusralv

**Comment by mateusralv** [comment]
ci:test:frontend:nocompile

**Comment by mateusralv** [comment]
continue: https://github.com/liferay-frontend/liferay-portal/pull/3064

**Pull request closed without merging**